[2025-02-02T15:57:29.354+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T15:57:29.368+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flights_pipeline.get_data_fly manual__2025-02-02T15:57:26.292766+00:00 [queued]>
[2025-02-02T15:57:29.378+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flights_pipeline.get_data_fly manual__2025-02-02T15:57:26.292766+00:00 [queued]>
[2025-02-02T15:57:29.380+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2025-02-02T15:57:29.408+0000] {taskinstance.py:2879} INFO - Executing <Task(_PythonDecoratedOperator): get_data_fly> on 2025-02-02 15:57:26.292766+00:00
[2025-02-02T15:57:29.415+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=94) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-02T15:57:29.418+0000] {standard_task_runner.py:72} INFO - Started process 126 to run task
[2025-02-02T15:57:29.417+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'flights_pipeline', 'get_data_fly', 'manual__2025-02-02T15:57:26.292766+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/zdag.py', '--cfg-path', '/tmp/tmpowe21u5b']
[2025-02-02T15:57:29.419+0000] {standard_task_runner.py:105} INFO - Job 15: Subtask get_data_fly
[2025-02-02T15:57:29.477+0000] {task_command.py:467} INFO - Running <TaskInstance: flights_pipeline.get_data_fly manual__2025-02-02T15:57:26.292766+00:00 [running]> on host d628bd12729b
[2025-02-02T15:57:29.595+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flights_pipeline' AIRFLOW_CTX_TASK_ID='get_data_fly' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T15:57:26.292766+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T15:57:26.292766+00:00'
[2025-02-02T15:57:29.597+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T15:57:31.804+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T15:57:31.825+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T15:57:31.827+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=flights_pipeline, task_id=get_data_fly, run_id=manual__2025-02-02T15:57:26.292766+00:00, execution_date=20250202T155726, start_date=20250202T155729, end_date=20250202T155731
[2025-02-02T15:57:31.889+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2025-02-02T15:57:31.930+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-02T15:57:31.936+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
